.gitignore

```diff
@@ -19,6 +19,7 @@ vendor
 !/mod/dashboard/
 !/mod/developers/
 !/mod/diagnostics/
+!/mod/discussion/
```

Comment:
I'd also prefer plural but is it worth breaking all URLs and third-party plugins that use the current way?